bright-ifit-purchase-validation.yaml

```diff
@@ -181,3 +215,20 @@ paths:
           description: 'Something went wrong when attempting to entitle the user or general internal server error'
       tags:
         - Validate Purchase Transaction
+  /v1/tv-app/users/{userId}/subscription-info:
+    get:
+      summary: Returns some specific user subscription info
+      description: Provides particular user subscription info for BrightAI to cater what is displayed in the TV App purchase products catalogue
+      responses:
+        '200':
+          description: '200'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UserSubscriptionInfoResponse'
+        '401':
+          description: 'Unauthorized token header'
+        '404':
+          description: 'User with the provided id was not found or User does not have an active subscription'
```

Comment:
> or User does not have an active subscription

I feel like this specific case should not be an error, it doesn't make sense to me at first glance. What if we added a field to the response that is something like `hasActiveSubscription`  That way we could just return `false` in that field and null in the other fields instead of returning an error. 

This obviously isn't a huge deal, but I think I do think it would make things better/more readable in the long run.

Reply:
If we go this route, an alternative, is to have only `hasActiveSubscription` be required and `sku`, `accountSource`, & `subscriptionManagementUrl` be optional and only returned when `hasActiveSubscription` is true.

Thoughts?

Reply:
Makes complete sense to me. 👍

Reply:
Sounds good to me! (I never got around to playing with [the stripe api](https://github.com/jeffy-ifit/ifit-brightai-tv-openapi-spec/pull/3#discussion_r641527527), but this seems pretty intuitive)